CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing
+
```

Comment:
Remove file, not needed

Reply:
<img width="552" alt="image" src="https://github.com/user-attachments/assets/01a82249-375c-4d11-b0fa-0caa68cd2a9f">